### ISSUE TITLE
Publish this awesome extension to Open-VSX marketplace

### ISSUE THREAD

**Issue by hsinyinfu** [mjcrouch/vscode-perforce#222]
Hi, this plugin is very useful and awesome for Perforce users like me.
Is it possible to publish this extension to Open-VSX to make more users have the opportunity to enjoy the extension?

[Open-VSX](https://open-vsx.org/) is an open source extension marketplace from Eclipse and it worked like the [Microsoft extension marketplace](https://marketplace.visualstudio.com/) embedded in Visual Studio Code.
The source project of Open-VSX on Github could be accessed from [here](https://github.com/eclipse/openvsx).
However, Visual Studio Code is a proprietary tool provided by Microsoft, and it is not allowed the open source version of it ([Code - OSS project](https://github.com/microsoft/vscode)) to access the Microsoft's extension marketplace.
Users of the binary built from Code - OSS open source project (e.g., [VSCodium project](https://vscodium.com/)) could not access the Microsoft extension marketplace according to the license but have to access other alternatives like Open-VSX.

It is very grateful if you would be willing to publish your extension to Open-VSX extension marketplace.
To publish the extension to Opev-VSX as the author of this extension, you could check the following Q&A from Open-VSX:
* https://www.eclipse.org/legal/open-vsx-registry-faq/#faq-4
* https://github.com/eclipse/openvsx/wiki/Namespace-Access

If you would like someone to help you to publish this to Open-VSX to save time for you, Open-VSX also provides a way for anyone who is not the author of the extension under [OSI-approved open source licenses](https://opensource.org/licenses) to register the extension, and they will fetch (mirror) extensions automatically and push these extensions to the marketplace with the Namespace reserved for the author to register in the future.
It would be my pleasure to help to create a PR to register this awesome extension if you thought this is a better way than publishing it by yourself.
Check the Oepn-VSX mirror project for more details.
* https://github.com/open-vsx/publish-extensions

**Comment by loujaybee** [comment]
Heads up: There are some repercussions with using publish-extensions as the CI for publishing, and it's preferred that extension owners publish assets themselves. We're trying to [make this a bit clearer in an update to the docs](https://github.com/open-vsx/publish-extensions/pull/483/files) currently. 

tl;dr;

1. Using publish extension will mark extensions with a marketplace warning, publishing via the extension owner does not
2. Publishing via the extension owner, is often better for stability (keeps build steps/history in one place, also)
3. It's easier/better for users to raise any bugs/features with extension owners rather than publish-extensions
